faculty/datasets/__init__.py

```diff
@@ -20,38 +20,18 @@
 import tempfile
 import io
 
-from botocore.client import ClientError
-
-from faculty.datasets import path, session
-from faculty.datasets.session import DatasetsError
+from faculty.session import get_session
+from faculty.context import get_context
+from faculty.clients.object import ObjectClient
+from faculty.datasets import path, transfer
+from faculty.datasets.path import DatasetsError
 
 
 # For backwards compatibility
 SherlockMLDatasetsError = DatasetsError
```

Comment:
Should we take the opportunity to remove this?

Reply:
@acroz and I spoke and decided not to remove this yet